apps/src/aichat/views/presentation/ModelCardRow.tsx

```diff
@@ -7,7 +7,7 @@ interface ModelCardRowProps {
   keyName: string;
   title: string;
   titleIcon: string;
-  expandedContent: string;
+  expandedContent: React.ReactNode;
```

Comment:
This is just the value of a model card field right? Should be either `string | string[]` or `ModelCardInfo[keyof ModelCardInfo]`?